__tests__/components/card.spec.js

```diff
@@ -1,6 +1,83 @@
-import {loadFixture, testVM} from '../helpers';
+import { loadFixture, testVM } from '../helpers'
 
 describe('card', async() => {
-    beforeEach(loadFixture('card'));
-    testVM();
-});
\ No newline at end of file
+    beforeEach(loadFixture('card'))
+    testVM()
+
+    // since our class test depends on the card block,
+    // run this test first
+    it('all examples should contain card block', async() => {
+        const { app: { $refs, $el } } = window
+
+        const refs = ['simple_card', 'standard_card', 'img_card', 'img_overlay_card']
+
+        refs.forEach((ref) => {
+            const childNodes = [...$refs[ref].$el.childNodes]
+            const cardBlock = childNodes
+                .find(el => el.classList && el.classList.contains('card-block'))
+
+            expect(cardBlock).toBeDefined()
```

Comment:
Well, [Array.prototype.find returns the element or undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find?v=control)
> The find() method returns the value of the first element in the array that satisfies the provided testing function. Otherwise undefined is returned.

So that should still be good, yes?